lib/auth/person.ex

```diff
@@ -22,6 +22,7 @@ defmodule Auth.Person do
     field :key_id, :integer
 
     has_many :statuses, Auth.Status
+    has_many :login_logs, Auth.LoginLog
```

Comment:
Login attempts should not "belong" to the `person`. 
They are a system log that we are storing for security reasons.
If the login is _successful_ we can show it to the person in the interface as:
"you are logged in from these 3 devices".
We don't need to use `Ecto` associations for that, unless there is some sort of benefit to it ... 💭